### PR TITLE
fix: automerge upgrade PRs workflow added

### PR DESCRIPTION
**Pull request by zubairshakoorarbisoft**

### CODE REVIEW

**Review by iamsobanjaved — APPROVED** [review]
LGTM, but just make sure to merge this after the related PR in the `openedx/.github` repo. Also make sure you squash commits before merging

**Comment by zubairshakoorarbisoft** [comment]
> LGTM, but just make sure to merge this after the related PR in the `openedx/.github` repo. Also make sure you squash commits before merging

Thanks for reviewing PR. I have made changes in openedx PR requested by you, please take a look into them when you have a moment.

**Pull request closed without merging**
